src/core/event.js

```diff
@@ -482,9 +483,19 @@ Candy.Core.Event = (function(self, Strophe, $) {
 			}
 
 			if(directInvite.length > 0) {
```

Comment:
I don't think we can assume that either will come from someone in our roster. A mediated invite will actually come from the MUC service itself.